int32 id
string type
string[] relations
---

